src/components/StatsScoringDialog.vue

```diff
@@ -1,96 +1,101 @@
 <template>
-  <v-dialog v-model="show">
+  <base-dialog
+    v-model="show"
+    title="Ranked Scoring"
+    :id="`rankedScoring`"
```

Comment:
This is a simple string so I think we can just remove the v-bind and make it a literal
```suggestion
    id="rankedScoring"
```